cypress/e2e/models/migration/applicationinventory/application.ts

```diff
@@ -155,9 +155,10 @@ export class Application {
     public static open(forceReload = false): void {
         const itemsPerPage = 100;
         if (forceReload) {
-            cy.visit(Application.fullUrl, { timeout: 35 * SEC }).then((_) =>
-                selectItemsPerPage(itemsPerPage)
-            );
+            cy.visit(Application.fullUrl, { timeout: 35 * SEC }).then((_) => {
+                cy.get("h1", { timeout: 100 * SEC }).should("contain", applicationInventory);
```

Comment:
I think a timeout of 100 secods is too much.

I think if a page takes more than 35 secs (the current timeout we have for other tests) to load the test should fail because something is wrong there

Reply:
we need to reduce the time and  let this test fail .
It will impact overall time of execution.

Reply:
Lot of tests are using application page , let's reduce it to 5 Sec